awx/main/analytics/collectors.py

```
@@ -227,7 +227,7 @@ def counts(since, **kwargs):
     counts['pending_jobs'] = models.UnifiedJob.objects.exclude(launch_type='sync').filter(status__in=('pending',)).count()
     if connection.vendor == 'postgresql':
         with connection.cursor() as cursor:
-            cursor.execute(f"select count(*) from pg_stat_activity where datname=\'{connection.settings_dict['NAME']}\'")
+            cursor.execute(f"select count(*) from pg_stat_activity where datname='{connection.settings_dict['NAME']}'")
```

Ruff (0.16.0)

[error] Possible SQL injection vector through string-based query construction

(S608)